### ISSUE TITLE
DashboardOverview Revamp

### ISSUE THREAD

**Issue by Apexal** [Apexal/late#548]
The current overview is not very useful as it is read-only (except for confirming): 
![image](https://user-images.githubusercontent.com/8422699/68950038-5b2fe180-0789-11ea-8198-57103019809d.png)

---

We should revamp it to allow for quick, useful interactions. For example, the Courseload section (#533) should listen one or two important upcoming assessments. The "currently schedule" section should *always* show and allow for immediate scheduling of a work block starting at the current time.

**Comment by bertrk** [comment]
- [x] design new dashboard 

- [ ] implement left side (courseload)

- [ ] implement right side (stats?)

**Comment by Apexal** [comment]
> From the previous issue:

The DashboardOverview is currently **bad**. This is my fault because I made it. The info it shows is either too hard to follow (the timeline) or too general to mean anything (the workload weight).

Instead of the current components there should be:
- [ ] something showing what is currently going on
	- if nothing at the moment, suggest assessments to schedule work blocks at the next 15 minute interval
	- if work block, offer to cut off at the next 15 minute interval OR extend by 15 minutes
- [ ] weight of **remaining current school week's** workload
	- should show pressing items like exams and high priority assignments
- [ ] keep the assessment confirm component
- [ ] a map with all the locations of the day's classes on it
- [ ] coursework suggestions (#366)
